dark/logic/tround_utils.py

```diff
@@ -1,23 +1,30 @@
 import os
 
+from dark.models.tournament import TournamentRound
 
-def execute_round():
+
+def execute_round(round_id):
+    tround = TournamentRound.objects.get(pk=round_id)
+    platform = tround.platform
     # save current working directory
     current_dir = os.getcwd()
+    if not current_dir.endswith('/DARK'):
+        current_dir = current_dir[:(current_dir.find('/DARK') + 5)]
+        os.chdir(current_dir)
+    # set directory for round logs
+    log_directory = f'round_results'
 
-    # set directory for queue logs
-    log_directory = 'round_results'
-
-    # change directory to gupb and execute queue
-    os.chdir('GUPB')
-    os.system('python -m gupb -l {}'.format(log_directory))
+    # change directory to gupb and execute round
+    os.chdir(f'{platform.name}_{round_id}')
+    os.system(f'python3 -m {platform.package_to_run} -l {log_directory}')
```

Comment:
jak wyżej: https://stackoverflow.com/questions/12332975/installing-python-module-within-code

Reply:
najlepiej to tworzyć venva dla każdej platformy: https://docs.python.org/3/library/venv.html